engine-comparison/dispatcher.sh

```diff
@@ -5,61 +5,96 @@
 . $(dirname $0)/../common.sh
 . ${SCRIPT_DIR}/common-harness.sh
 
+
 build_engine() {
-  # [[ ! -e ~/$FENGINE_CONFIGS_DIR/$1 ]] && echo "cant do" && exit 1
-  # All of this code will probably be in a separate script
 
   FENGINE_CONFIG=$1
-
+  [[ ! -e $FENGINE_CONFIG ]] && echo "File DNE" && exit 1
   echo "Building $FENGINE_CONFIG"
   FENGINE_NAME=$(basename $FENGINE_CONFIG)
-  ENGINE_DIR=$WORK/fuzz-engines/${FENGINE_NAME}
-  rm -rf $ENGINE_DIR
-  mkdir $ENGINE_DIR
-
-  # Fuzzing activities go here
+  FENGINE_DIR=$WORK/fengine-builds/${FENGINE_NAME}
+  rm -rf $FENGINE_DIR
+  mkdir $FENGINE_DIR
+
+  . $FENGINE_CONFIG
+  # Build either engine
+
+  if [[ $FUZZING_ENGINE == "libfuzzer" ]]; then
+    echo "Making a version of libfuzzer"
+    svn co http://llvm.org/svn/llvm-project/llvm/trunk/lib/Fuzzer $FENGINE_DIR
```

Comment:
It's true that this is a tradeoff; I think I recall discussing this @kcc, with the conclusion that we prefer to use tip-of-tree libfuzzer, but on the other hand an older/stable version of clang. 

Thinking this a bit further, I think that this harness will be used primarily to develop improvements to the current version of libfuzzer, and pulling tip-of-tree is logical from that perspective.